### PR TITLE
[full-ci] Update to ODS 9.3.0

### PR DESCRIPTION
**Pull request by kulmann**
## Description
This PR updates ODS to v9.3.0, including required changes.

## How Has This Been Tested?
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [x] fix tests that use the `show details` button to enforce showing the right sidebar. Probably need to use the contextmenu for that now. Or a combination of selecting a resource and then toggling the sidebar.#
- [x] write changelog item

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/18550/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

**Comment by ownclouders** [comment]
Results for <strong>oC10Trashbin</strong> https://drone.owncloud.com/owncloud/web/18596/36/1
The following scenarios passed on retry:
- webUITrashbinFilesFolders/trashbinFilesFolders.feature:38

**Comment by ownclouders** [comment]
Results for <strong>oC10Basic</strong> https://drone.owncloud.com/owncloud/web/18697/7/1
The following scenarios passed on retry:
- webUIPrivateLinks/accessingPrivateLinks.feature:25

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18714/24/1
The following scenarios passed on retry:
- webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:114

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/18740/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

**Comment by kulmann** [comment]
cc @elizavetaRa

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/18745/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18760/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersToRootSharingIndicator/shareWithUsers.feature:98

**Review by LukasHirt — APPROVED** [review]

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 1]
```diff
@@ -1018,18 +1015,21 @@ Then(
 )
 
 When(
-  'the user opens the actions sidebar panel of file/folder {string} on the webUI',
-  async function(resource) {
-    await client.page.FilesPageElement.filesRow().openFileActionsMenu(resource)
+  /^the user opens the actions sidebar panel of (file|folder) "([^"]*)" on the webUI$/,
```
Comment (pascalwengerter) [inline tests/acceptance/stepDefinitions/filesContext.js:1018]: O.o

**Review by pascalwengerter — APPROVED** [review]
LGTM, thanks for all the refactoring & code deletion!

File: tests/acceptance/stepDefinitions/filesContext.js (hunk 1 above)
Comment (kulmann) [reply tests/acceptance/stepDefinitions/filesContext.js:1018]: just an improvement to be strict with the `file` or `folder` element type. ;-)

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersRoot2</strong> https://drone.owncloud.com/owncloud/web/18772/30/1
The following scenarios passed on retry:
- webUISharingInternalUsersToRootShareWithPage/shareWithUsers.feature:79

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18773/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersToRootSharingIndicator/shareWithUsers.feature:98

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18773/24/1
The following scenarios passed on retry:
- webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:114

**Comment by ownclouders** [comment]
Results for <strong>oC10CreateDelete</strong> https://drone.owncloud.com/owncloud/web/18773/9/1
The following scenarios passed on retry:
- webUIDeleteFilesFolders/deleteFilesFolders.feature:59

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternalUsers1</strong> https://drone.owncloud.com/owncloud/web/18773/55/1
The following scenarios passed on retry:
- webUISharingInternalUsers/shareWithUsers.feature:337

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18775/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersSharingIndicator/shareWithUsers.feature:154

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingPublicManagement</strong> https://drone.owncloud.com/owncloud/web/18775/34/1
The following scenarios passed on retry:
- webUISharingPublicManagement/publicLinkIndicator.feature:110

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/18775/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

**Pull request merged**